github-action-build/setup.sh

```
@@ -28,24 +28,26 @@ if [ ! -d "$TEST_HOME" ]; then
+wget "https://github.com/wso2/financial-services-accelerator/releases/download/v3.3.0/wso2-obiam-accelerator-3.3.0.zip" -O wso2-obiam-accelerator-3.3.0.zip
+unzip $TEST_HOME/wso2-obiam-accelerator-3.3.0.zip -d $TEST_HOME/wso2is-6.1.0
 
-wget "https://atuwa.private.wso2.com/WSO2-Products/open-banking-accelerators/3.0.0/wso2-obam-accelerator-3.0.0.zip" -O $TEST_HOME/wso2-obam-accelerator-3.0.0.zip
-unzip $TEST_HOME/wso2-obam-accelerator-3.0.0.zip -d $TEST_HOME/wso2am-4.2.0
+wget "https://github.com/wso2/financial-services-accelerator/releases/download/v3.3.0/wso2-obam-accelerator-3.3.0.zip" -O wso2-obam-accelerator-3.3.0.zip
+unzip $TEST_HOME/wso2-obam-accelerator-3.3.0.zip -d $TEST_HOME/wso2am-4.2.0
 
-wget "https://atuwa.private.wso2.com/WSO2-Products/open-banking-accelerators/3.0.0/wso2-obbi-accelerator-3.0.0.zip" -O $TEST_HOME/wso2-obbi-accelerator-3.0.0.zip
-unzip $TEST_HOME/wso2-obbi-accelerator-3.0.0.zip -d $TEST_HOMER/wso2si-4.2.0
+wget "https://github.com/wso2/financial-services-accelerator/releases/download/v3.3.0/wso2-obbi-accelerator-3.3.0.zip" -O wso2-obbi-accelerator-3.3.0.zip
+unzip $TEST_HOME/wso2-obbi-accelerator-3.3.0.zip -d $TEST_HOMER/wso2si-4.2.0
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)